server/routes/serviceProviderReferrals/editSessionView.ts

```diff
@@ -134,4 +140,104 @@ export default class EditSessionView {
       ],
     }
   }
+
+  private get otherLocationFirstAddressLineInputArgs(): InputArgs {
```

Comment:
would be good to have these in a reusable location so we can use for supplier assessments

Reply:
Sure, that would be nice.

I don't think we have used this specific pattern before except possibly with LayoutView. The other shared Views have an inheritance relation rather than composition.

So my plan would be to  create an `addressFormView.ts` and an `addressFormPresenter.ts`. Does this sound agreeable to you?